README.md

```diff
@@ -16,6 +16,8 @@ $ embulk gem install embulk-parser-none
 ## Configuration
 
 - **column_name**: A column name which this plugin outputs (string, default: "payload")
+- **line_delimiter_recognized**: `CRLF`, `LF` or `CR`. If specified, only the character is recognized as line delimiter. Otherwise, all of them are recognized (enum, default: `null`)
+- **charset**: Character encoding eg. `ISO-8859-1`, `UTF-8` (string, default: `UTF-8`)
```

Comment:
I'm not very familiar with `embulk-parser-none`, but does it need these configurations?

(If not, just giving default values to `LineDecoder` without making them configurable is an option.)

Reply:
Revert this change. (Just removed README)
IMO, a user will not use `line_delimiter_recognized` much. But some users may want to specify the `charset`.